docs/source/testing-and-debugging.md

```diff
@@ -43,71 +43,79 @@ from boa3.boa3 import Boa3
 Boa3.compile_and_save('path/to/your/file.py', debug=True)
 ```
 
-## Neo Test Runner
+## boa test constructor
 
 ### Downloading
 
-Install [Neo-Express](https://github.com/neo-project/neo-express#installation) and [Neo Test Runner](https://github.com/ngdenterprise/neo-test#neo-test-runner).
+Install [boa-test-constructor](https://pypi.org/project/boa-test-constructor/) with pip. We use this extension to run an isolated
+test environment for smart contracts with a neo-go node. When installing boa-test-constructor, [neo-mamba](https://dojo.coz.io/neo3/mamba/index.html)
+will be installed too.
 
 ### Testing
 
-Before writing your tests, make sure you have a Neo-Express network for local tests.
-If you do not yet have a local network, open a terminal and run `neoxp create`.
-Please refer to [Neo-Express documentation](https://github.com/neo-project/neo-express/blob/master/docs/command-reference.md#neoxp-create)
-for more details of how to configure your local network. 
+Create a Python Script, import the `SmartContractTestCase` class, and create a test class that inherits it. To set up
+the test environment, you'll need to override the `setUpClass` method from `SmartContractTestCase`. This method is 
+synchronous, so if you need to set up asynchronous tasks, you can create another async method and use it int the 
+`asyncio.run` method from `asyncio`. Common operations would be: creating accounts, deploying the smart contract, 
+selecting your "main" smart contract, and transferring GAS to the new accounts.
```

Comment:
it is worth mentioning why the tests must be async:
the tests use a local chain to run the invocations and the communication with the chain is asynchronous, so anything that needs to interact with it (deploy, token transfers, vote, etc) must be awaited